### PR TITLE
Music playlist fixes

### PR DESCRIPTION
**Pull request by wissog**
Fix the title above the playlist
Make the songs automatic paly one after another

still need to fix the click on playlist item

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const routes = [
     component: LessonCollection,
     options: { ssrData: ssrDataLoaders.lessonsCollectionPage }
   },
-  { path: ':routeType/:tab/cu/:id', component: PlaylistItemPage, options: { ssrData: ssrDataLoaders.cuPage } },
+  // { path: ':routeType/:tab/cu/:id', component: PlaylistItemPage, options: { ssrData: ssrDataLoaders.cuPage } },
```
Comment (davgur) [inline src/routes.js:69]: @wissog ?

File: src/components/Pages/PlaylistCollection/Page.js [hunk 2]
```diff
@@ -65,9 +65,11 @@ const PlaylistCollectionPage = ({ collection, nextLink = null, prevLink = null,
 
   const handleSelectedChange = useCallback(nSelected => {
     if (nSelected !== selected && playlist?.items && playlist?.items[nSelected]) {
-      history.push(`/${uiLanguage}${playlist.items[nSelected].shareUrl}`);
+      // console.log('share url:', playlist.items[nSelected].shareUrl)
+      // history.push(`/${uiLanguage}${playlist.items[nSelected].shareUrl}`);
```
Comment (davgur) [inline src/components/Pages/PlaylistCollection/Page.js:69]: @wissog why you remove it? is bug that it was fixe not reproduce? cant find how you fixe it

File: src/components/Pages/PlaylistCollection/Container.js [hunk 3]
```diff
@@ -66,6 +56,24 @@ const PlaylistCollectionContainer = ({ cId, t, cuId }) => {
   }, [cId, dispatch, collection, wipMap.collections]);
 
   useEffect(() => {
+    const fetchWindow = () => {
```
Comment (davgur) [inline src/components/Pages/PlaylistCollection/Container.js:59]: @wissog you create functions inside of effect?

Comment (wissog) [reply src/components/Pages/PlaylistCollection/Container.js:59]: you have to if you use that function inside the effect

File: src/routes.js (hunk 1 above)
Comment (wissog) [reply src/routes.js:69]: this causes Article Page to be displayed like a video item...
and i didn't find its contribution

File: src/components/Pages/PlaylistCollection/Page.js (hunk 2 above)
Comment (wissog) [reply src/components/Pages/PlaylistCollection/Page.js:69]: this causes to open the next item in a new page instead of remaining in the playlist

**Review by davgur — APPROVED** [review]

**Pull request merged**